libs/cart/src/drivers/magento/queries/list-shipping-methods.ts

```diff
@@ -8,6 +8,7 @@ import { availableShippingMethodFragment } from './fragments/public_api';
 export const listShippingMethods = (extraCartFragments: DocumentNode[] = []) => gql`
   query ListShippingMethods($cartId: String!) {
     cart(cart_id: $cartId) {
+      id
```

Comment:
just curious. Why was the id added to the query?

Reply:
Apollo would refuse to write the query result to the store without an ID. I'm not sure why it doesn't do this under any other circumstances.